### ISSUE TITLE
netcdf4 should be declared as a dependency

### ISSUE THREAD

**Issue by yurivict** [materialsproject/pymatgen#1386]
Simple import of this module fails:
```
No module named netCDF4

Please install netcdf4 with `conda install netcdf4`
If the conda version does not work, uninstall it with `conda uninstall hdf4 hdf5 netcdf4`
and use `pip install netcdf4`
  and use `pip install netcdf4`""" % str(exc))
```

You need to add netcdf4 to the package requirements,

**Comment by shyuep** [comment]
What were you importing? Just pymatgen?

**Comment by yurivict** [comment]
Yes. Once it is directly required by pymatgen, it should depend on it.

**Comment by shyuep** [comment]
I don't think pymatgen requires it. Pls provide the version of pymatgen you are using.

**Comment by yurivict** [comment]
> pymatgen-2019.1.24/pymatgen/io/abinit/netcdf.py:`import netCDF4` failed with the following error:

**Comment by shyuep** [comment]
This is a problem with the abinit module of pymatgen. Therefore, netcdf4 is required only for the abinit package. It is listed under optional dependencies. If you simply just use the rest of pymatgen, there is no requirement for netcdf4.

**Comment by gmatteo** [comment]
The netcdf library is required to read the netcdf output files produced by Abinit.
The library is one of the *optional* dependencies of pymatgen listed in requirements-optional.txt
To install netcdf for python, use:

```
pip install -r requirements-optional.txt
```

or alternatively use conda and the command:

```
conda install netcdf4 -c conda-forge
```

Alternatively, you may want to install the abipy package following the instructions reported [here](https://github.com/abinit/abipy#getting-abipy).
Abipy  depends on netcdf and pymatgen so the installation script will automatically install 
these two dependencies.
